src/main/java/core/basesyntax/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.dao.jdbc;
+
+import core.basesyntax.dao.ManufacturerDao;
+import core.basesyntax.exception.DataProcessingException;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Manufacturer;
+import core.basesyntax.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String queryInsert = "INSERT INTO manufacturers "
+                + "(manufacturer_name, manufacturer_country) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement insert = connection.prepareStatement(queryInsert,
+                         Statement.RETURN_GENERATED_KEYS)) {
+            insert.setString(1, manufacturer.getName());
+            insert.setString(2, manufacturer.getCountry());
+            insert.execute();
+            ResultSet resultSet = insert.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert manufacturer " + manufacturer, e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String query = "SELECT * FROM manufacturers WHERE manufacturer_id=? AND deleted=FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement selectById = connection.prepareStatement(query)) {
+            selectById.setLong(1, id);
+            ResultSet resultSet = selectById.executeQuery();
+            Manufacturer manufacturer = new Manufacturer();
```

Comment:
```suggestion
            Manufacturer manufacturer = null;
```
you never return empty optional in current impl